Dockerfile

```diff
@@ -35,9 +63,15 @@ COPY package.json package-lock.json /usr/src/app/
 RUN npm ci --unsafe-perm --production && npm cache clean --force 2>/dev/null
 
 COPY --from=builder /usr/src/app/build /usr/src/app/build
+COPY --from=gobuild /usr/bin/node_exporter /bin/
```

Comment:
i think you should copy this into `/usr/bin`, to stay closer to https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard